source/viz.md

```diff
@@ -805,23 +848,23 @@ you can clearly see the mutated output from the table.
 ```
 
 ```{code-cell} ipython3
+canadian_population = 35_151_728
```

Comment:
this is a nice way to format a number, but have we explained to students that we can do this? This should probably appear in an earlier chapter if not...or maybe here if it's the first time it's relevant / used

Reply:
Added a one sentence explanation